linux-cachyos-rc/PKGBUILD

```
@@ -472,6 +479,12 @@ prepare() {
 
     echo "Selecting '$_hugepage' TRANSPARENT_HUGEPAGE config..."
 
+    # Enable Clang AutoFDO
+    # Add additonal check if Thin or Full LTO is enabled otherwise die
+    if [ "$_autofdo" ] || [ -n "$_autofdo_use" ]; then
```

```suggestion
    if [ -n "$_autofdo" ] || [ -n "$_autofdo_use" ]; then
```
We need to follow some consistent check style.